lib/dotcom_web/components/system_status/status_label.ex

```diff
@@ -22,11 +22,14 @@ defmodule DotcomWeb.Components.SystemStatus.StatusLabel do
     ~H"""
     <span class={[status_classes(@status), "flex items-center gap-2"]}>
       <.status_icon status={@status} />
-      {@rendered_prefix} {description(@status, @plural)}
+      {@rendered_prefix} {expectation(@status, @prefix)} {description(@status, @plural)}
```

Comment:
Non-blocking: I think that this would be easier to understand if the whole decision tree of `"Delay"` versus `"Expect Delays"` versus some other status like `"Shuttle"` happened purely inside the `description` function. That means that `description` would need to just take the whole glob of data at this point - either `defp description(@status, @prefix, @plural)` or just `defp description(@status_entry)`, but we wouldn't have the distinction between `"Delay"` and `"Expect Delay"` be a top-level concern.

I'm happy to pair on this, and/or treat it as a follow-up, since it does clearly work, and the feature here is worth getting merged, but I would like to clean this up at some point.

Reply:
> happened purely inside the description function

Ha that's how I started! Though I wasn't clever enough to try `description(@status, @prefix, @plural)`, which I like here -- will try that